_layouts/blog.html

```diff
@@ -1,16 +1,22 @@
 ---
-layout: default
+layout: page
```

Comment:
My idea was to have a nice splash image to showcase art made in mypaint on the landing page, and to keep the other pages simple.